deploy/contents/install/app/install-utils.sh

```diff
@@ -578,26 +578,28 @@ function enable_all_services {
 }
 
 function is_data_is_available_in_point_in_time_configuration {
-  local point_in_time_configuration_dir="${1}"
-  local point_in_time_configuration_module="${2}"
-  local pitc_metadata_file="$point_in_time_configuration_dir/_pitc.csv"
-
-  while IFS="," read -r data module_file exit_code; do
-  if [ "$point_in_time_configuration_module" == "$data" ] && [ -s "$module_file" ] && [ $exit_code -eq 0 ]; then
-     echo $module_file
-     return 0
+  local point_in_time_configuration_module="${1}"
+
+  if [ $CP_POINT_IN_TIME_CONFIGURATION_DIR ]; then
+     local pitc_metadata_file="$CP_POINT_IN_TIME_CONFIGURATION_DIR/_pitc.csv"
+
+     while IFS="," read -r data module_file exit_code; do
+     if [ "$point_in_time_configuration_module" == "$data" ] && [ -s "$CP_POINT_IN_TIME_CONFIGURATION_DIR/$module_file" ] && [ $exit_code -eq 0 ]; then
+        echo $module_file
```

Comment:
lets return here full path to the file as `"$CP_POINT_IN_TIME_CONFIGURATION_DIR/$module_file" `